.github/pull_request_template.md

```diff
@@ -19,20 +19,18 @@ Closes #ISSUE
 
 ### Checklist
 
-Before creating a PR, run through this checklist and mark each as complete.
-
-- [ ] I have read the [contributing guidelines](/CONTRIBUTING.md)
-- [ ] I have signed the [F5 Contributor License Agreement (CLA)](https://github.com/f5/.github/blob/main/CLA/cla-markdown.md)
-- [ ] If applicable, I have added tests that prove my fix is effective or that my feature works
-- [ ] If applicable, I have checked that any relevant tests pass after adding my changes
-- [ ] I have updated any relevant documentation ([`README.md`](/README.md) and [`CHANGELOG.md`](/CHANGELOG.md))
-- [ ] I have rebased my branch onto main
-- [ ] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork
-- [ ] If the change involves:
-  - Code
-  - Anything that resembles Personally identifying information (PII)
-    - Make sure to use placeholders such as `<username>` in place of PII
-  - URLs (watch for [typosquatting](https://support.microsoft.com/en-us/topic/what-is-typosquatting-54a18872-8459-4d47-b3e3-d84d9a362eb0))
-  - Significant new/revised content
-    
-  In these cases, the change will require at least two (2) approvals before merging
+Before merging a pull request, run through this checklist and mark each as complete.
+
+- [] I have read the [contributing guidelines](/CONTRIBUTING.md)
+- [] I have signed the [F5 Contributor License Agreement (CLA)](https://github.com/f5/.github/blob/main/CLA/cla-markdown.md)
+- [] I have ensured that documentation content adheres to [the style guide](/templates/style-guide.md)
+- [] If the change involves potentially sensitive changes, I have assessed the possible impact
+- [] If applicable, I have added tests that prove my fix is effective or that my feature works
+- [] If applicable, I have checked that any relevant tests pass after adding my changes
+- [] I have updated any relevant documentation ([`README.md`](/README.md) and [`CHANGELOG.md`](/CHANGELOG.md))
+- [] I have rebased my branch onto main
+- [] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork
+
+Potentially sensitive changes include anything involving code, personally identify information (PII), live URLs or significant amounts of new or revised documentation.
```

Comment:
maybe:

```suggestion
- [] Pay special attention to the following types of changes: code, personally identify information (PII), URLs (watch for typosquatting), or significant amounts of new or revised documentation.
```

Reply:
The sentence is intended to support the existing item on line 27: this makes it somewhat redundant.

I'm banking on the idea that we give better explanations of these issues in the style guide, as reducing the amount of reading fatigue on a template some people might be exposed to daily feels paramount.

Reply:
> The sentence is intended to support the existing item on line 27: this makes it somewhat redundant.

- I'd be good if it were a sub-item of line 27. When our security engineers looked at this repo, they wanted us to pay special attention to these items.

Reply:
While I'd love to see automation, I think this is beyond anything we could put in our style guide. (If you can show me how, I'd appreciate it!.)

Reply:
I'm not really sure why automation is part of this discussion.